onnx/defs/tensor/defs.cc

```diff
@@ -701,8 +701,7 @@ ONNX_OPERATOR_SCHEMA(SpaceToDepth)
     .Attr(
         "blocksize",
         "Blocks of [blocksize, blocksize] are moved.",
-        AttributeProto::INT,
-        OPTIONAL)
+        AttributeProto::INT)
```

Comment:
@houseroad / @bddppq / @ebarsoum , are we all ok to change to the "blocksize" from "optional" to "required" please? or actually this should be "required" and was a spec bug?

Reply:
@linkerzhang I agree this should be required and should be considered as a bug (instead of a breaking change).

Reply:
I think it's spec bug. Usually, users should aware what he is doing, :-)